spec/requests/api/v2/profiles_controller_update_spec.rb

```diff
@@ -0,0 +1,205 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::ProfilesController do
+  let(:response_json) { JSON.parse(response.body) }
```

Comment:
I keep wondering whether we want to move to move this into the rails helper or something? It's so common in all of our request specs